lms/validation/_api.py

```diff
@@ -132,3 +137,31 @@ def validate(self, data, **_kwargs):
             raise ValidationError(
                 "both bookID and cfi are mandatory with type is 'vitalsource'"
             )
+
+
+class APISyncBlackboardSchema(PyramidRequestSchema):
```

Comment:
```suggestion
class APIBlackboardSyncSchema(PyramidRequestSchema):
```

Ideally I think it should be named `BlackboardSyncAPISchema`. In speaking we call it the "Blackboard sync API" and we'd call the schema the "Blackboard sync API schema", so `BlackboardSyncAPISchema`

But there's something weird going on with the naming of the schemas in this file: `APIRecordSpeedgraderSchema`, `APIReadResultSchema`, `APIRecordResultSchema`, `APICreateAssignmentSchema`, etc. If it's the "create assignment API" then I'd think it'd be called `CreateAssignmentAPISchema`. I'm not sure why we keep putting `API` at the start.

But if we want to be consistent with those existing names we should put `API` at the start as well